validate_roles.py

```diff
@@ -3,15 +3,23 @@
 import json
 from termcolor import cprint
 
-REQUIRED_KEYS = {"role", "url", "sub-role", "lead", "deputy", "role-head",
-                 "role-subhead", "description", "sub-description"}
+REQUIRED_KEYS = {"role", "url", "role-head", "responsibilities"}
 
-LIST_KEYS = {"sub-role", "lead", "deputy", "role-head", "role-subhead",
-             "description", "sub-description"}
 
-STRING_KEYS = {"role", "url"}
+def assert_list(i, key, value):
+    if isinstance(value, list):
+        return 0
+    else:
+        cprint(f"   ERROR: Key \"{key}\" for role #{i} should be a list but instead is a {type(value)}", file=sys.stderr, color='red')
+        return 1
+
 
-LENGTH_MATCH_KEYSETS = [{"sub-role", "lead", "deputy"}]
+def assert_string(i, key, value):
```

Comment:
```suggestion
def assert_is_string(i, key, value):
```
(goes with the other suggestion)